concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -44,9 +44,236 @@ meta:
     curl-ssl: &curl-ssl-image-resource
       type: docker-image
       source:
-        repository: governmentpaas/curl-ssl
-        tag: 54cc3c9f49f9032e46dd4538c626f2533bf90a94
+        repository: tnwhitwell/curl-ssl
+        tag: 36db896bf1e6dd8f4034b539f814e1622a583966
```

Comment:
I think this was to pull in `jq` and we should be able to use our own image updated with https://github.com/alphagov/paas-docker-cloudfoundry-tools/pull/173.